rclcpp/include/rclcpp/node_interfaces/node_base_interface.hpp

```diff
@@ -128,6 +128,13 @@ class NodeBaseInterface
   const std::vector<rclcpp::CallbackGroup::WeakPtr> &
   get_callback_groups() const = 0;
 
+  using CallbackGroupCallback = std::function<void (rclcpp::CallbackGroup::SharedPtr)>;
```

Comment:
nitpick: another naming thing, when I think of "callback" (especially in the context of ROS) I think of asynchronous and persistent, but that's purely subjective. This is, however, just a function-like thing used in the scope of the method, so I'd have done `CallbackGroupFunction` or something like that. `std::for_each` calls it a "unary function", so it also aligns with that better too:

https://en.cppreference.com/w/cpp/algorithm/for_each

Reply:
Yeah, I didn't love the name `CallbackGroupCallback` either.  I've opted for `CallbackGroupFunction`, so as not to make it too long.  Let me know what you think.